src/include/ipc4/module.h

```diff
@@ -334,6 +334,16 @@ struct ipc4_module_set_d0ix {
 	} data;
 } __attribute__((packed, aligned(4)));
 
+struct ipc4_dx_state_info {
```

Comment:
Isn't D3 a global message since it impacts whole FW or do we send a D3 message to the base FW module to put whole DSP in D3 ?

Reply:
This is a message to base FW module, but it affects entire DSP.